README.md

```diff
@@ -13,16 +13,22 @@ layout:
     visible: true
 ---
 
-# Kontrol
-
 <div data-full-width="true">
 
 <figure><img src=".gitbook/assets/kontrol logo yellow.png" alt=""><figcaption></figcaption></figure>
 
 </div>
 
-[**Kontrol**](https://github.com/runtimeverification/kontrol) combines **KEVM** and [Foundry](https://book.getfoundry.sh/) to grant developers the ability to perform [formal verification](https://en.wikipedia.org/wiki/Formal\_verification) without learning a new language or tool. This is especially useful for those who are not verification engineers. Additionally, developers can leverage Foundry test suites they have already developed and use symbolic execution to increase the level of confidence.
+# Kontrol
+
+Struggling to kontrol your smart contracts behave as intended, even after thorough unit testing, fuzzing, and other validation procedures? Frustrated with finding bugs in your code but unable to prove their absence?
+
+[**Kontrol**](https://github.com/runtimeverification/kontrol) is designed for you!
+
 
-[**KEVM**](https://github.com/runtimeverification/evm-semantics) is a tool that enables formal verification of smart contracts on the [Ethereum](https://ethereum.org/en/) blockchain. It provides a mathematical foundation for specifying and implementing smart contracts. Developers can use **KEVM** to rigorously reason about the behavior of their smart contracts, ensuring correctness and reducing the likelihood of vulnerabilities in the contract code.
+- **Open Source**: Kontrol is open source under the [BSD-3 Clause License](https://github.com/runtimeverification/kontrol/blob/master/LICENSE). This gives you full Kontrol to use, modify, and redistribute it according to your needs.
+- **Easy to Use**: Take advantage of the languages and tests you’re already familiar with. Kontrol enables developers to write formal specifications as [Foundry](https://book.getfoundry.sh/) test suites in [Solidity](https://soliditylang.org). It allows developers to reuse their existing test suites for formal verification. Even if you're not a verification engineer, Kontrol lets you formally verify your contracts in minutes.
```

Comment:
I wouldn't really promise to verify a contract in minutes, because in a lot of real-world cases that may not happen.

```suggestion
- **Easy to Use**: Kontrol gives you the advantage of using languages and testing frameworks you’re already familiar with. Kontrol enables developers to write formal specifications as [Foundry](https://book.getfoundry.sh/) tests in [Solidity](https://soliditylang.org). In addition, this allows developers to reuse their existing test suites for formal verification.
```

Reply:
Thank you! I've committed your suggestion.